### DISCUSSION TITLE
Integrate RTLCSS.com ?

### DISCUSSION THREAD

**Discussion by aneesdev** [tailwindlabs/tailwindcss#9857] [Ideas]
Hi everyone,

I know tailwind gives me `rtl` utilities like `rtl:mr-3` etc. I was wondering if some how it is possible that I just style my site with just normal ltr tailwind e.g `mr-3` etc. and upon building when tailwind makes the final `style.css` file it also generates a `style-rtl.css` file using [rtlcss](https://github.com/MohammadYounes/rtlcss)?

@adamwathan What do you think?

**Comment by Palmieri78759** [comment]
I think it can, But I can't give a definitive answer as I haven't worked with Angular-CLI.

Since rtlcss is a postcss plugin, I suggest searching for integrating postcss with angular instead.
[PaybyPlateMa Pay Online
](https://www.paybyplatema.us/)
